### PR TITLE
[UI] Replace spinner icon

### PR DESCRIPTION
**Pull request by andersonjeccel**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🔴🟢
| Deprecations?                          | 🔴🟢
| BC breaks? (use the c.x branch)        | 🔴🟢
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 

## Description

This PR replaces the spinner and changes JS to work with new classes.
Potentially unstable, review carefully.

---
### 📋 Steps to test this PR:

1. Open the Dashboard
2. Open your browser's dev tools
3. Add the tab Network conditions
4. Set network throttling to Offline
5. In Mautic, change the date range in the dashboard and click Save
6. See the loader

### CODE REVIEW

File: app/bundles/CoreBundle/Assets/js/1.core.js [hunk 1]
```diff
@@ -390,51 +390,50 @@ var Mautic = {
...
                     }
                 }
                 mQuery(el).removeClass();
-                mQuery(el).addClass('fa fa-spinner fa-spin ' + identifierClass + appendClasses);
+                mQuery(el).addClass('ri-loader-line ri-spin ' + identifierClass + appendClasses);
+                mQuery(el).removeClass();
+                mQuery(el).addClass('ri-loader-line ri-spin ' + identifierClass + appendClasses);
             }
-        }
-    },
```
Comment (escopecz) [inline app/bundles/CoreBundle/Assets/js/1.core.js]: Is this really working for you? I'm getting syntax error
Screenshot 2024-07-12 at 10 17 15

The steps to test could be that throttling is set to slow 3G in dev tools and then change the date range in the dashboard to obfuscate the cache and reload the widgets.

Comment (andersonjeccel) [reply app/bundles/CoreBundle/Assets/js/1.core.js:423]: How can I check in a practical scenario?
Where can I trigger this loading icon?

Comment (andersonjeccel) [reply app/bundles/CoreBundle/Assets/js/1.core.js]: Thanks, it's working now!

**Review by escopecz — APPROVED** [review]
It's working now. I tested on the dashboard.
Before:
Screenshot 2024-07-15 at 11 24 23

After:
Screenshot 2024-07-15 at 11 22 03

**Review by PatrickJenkner — APPROVED** [review]
Works and looks good 👍

**Pull request merged**